docker/setup-local-environment.sh

```diff
@@ -7,7 +7,6 @@ cd "${POCS}"
 
 echo "Building local panoptes-pocs:latest"
 docker build \
-    --quiet \
```

Comment:
Can we add this back if it is working now or is it helpful to see the output?  I was just worried it was too much.

Reply:
I found it helpful in case the build were to throw any errors, but I can add it back.